### ISSUE TITLE
Feature request: Individual database settings

### ISSUE THREAD

**Issue by dotWee** [keepassium/KeePassium#64]
**Describe a feature you'd like**
It would be nice to have database specific settings. 

**Additional context**
Examples:
- **Database A** locks automatically after 5 minutes, **database B** after 60 minutes. **Database C** doesn’t utilise timeout locks.
- **Database A** has it’s clipboard timeout set to 60 seconds, **database B** clears the clipboard after 10 seconds.

**Additional idea**
Persisting KeePassium’s database settings inside the actual KeePass database as backup/restore source?

**Comment by keepassium** [comment]
Absolutely! Per-database settings are also necessary for some other features (such as read-only DBs).

Optionally persisting them in the database itself is also a great idea, thanks!

**Comment by Woidla27** [comment]
I like that idea!
Would apreciate to configure touch id per database too.

**Comment by dotWee** [comment]
**Additional Idea:**

Maybe `No Sort` should allow changeable entry positions, for example using drag'n drop.


Example: Currently it's not possible to move the `DatabaseA` at the bottom to the top. If the user prefers a custom sorting, it would be useful to allow individual positioning.

**Comment by keepassium** [comment]
Thanks, Lukas. Reordering items manually is on the list, but deep in the "nice to have" territory. 

Individual database settings have been 80% done since December — yet it is August already, and they are still 80% done...

**Comment by pardahlman** [comment]
I'm really looking forward to this feature, too 🌟  Any update as to when we can expect to see this released?

**Comment by keepassium** [comment]
@pardahlman , nearing the top of the queue, should probably be done during the summer.
